semaphore/integration_tests.py

```diff
@@ -0,0 +1,119 @@
+#/usr/bin/env python
+from datetime import date, time, datetime, timedelta
+from random import randrange, getrandbits
+import json
+import requests
+
+SPLUNK_HOST = "https://mbta.splunkcloud.com:8089"
+SAVED_SEARCH = "Top 10 trip plans %2B 10 latest trip plans"
+OTP_DEV = "https://dev.otp.mbtace.com"
+OTP_PROD = "https://prod.otp.mbtace.com"
+
+PREDEFINED_PLANS = [
+    # Airport shuttles 1
```

Comment:
why not include other options in the `PREDEFINED_PLANS`? e.g. preferred modes, arrive_by, walk reluctance?

Reply:
we of course can do it, but.. 
1) In 98% of cases customers are looking for _all_ modes of transpiration:
![image](https://user-images.githubusercontent.com/45011335/61385344-3fbad580-a880-11e9-89d4-94e14bdc16dc.png)
2) walk reluctance is always the same, it doesn't change;
3) we already set arrive_by to true/false randomly to increase test coverage. 

All in all, [yagni](https://www.martinfowler.com/bliki/Yagni.html).

Reply:
:+1: